vm/message/src/signed_message.rs

```diff
@@ -124,6 +124,34 @@ impl Cbor for SignedMessage {
     }
 }
 
+#[cfg(test)]
+impl quickcheck::Arbitrary for SignedMessage {
+    fn arbitrary(g: &mut quickcheck::Gen) -> Self {
+        const DUMMY_SIG: [u8; 1] = [0u8];
+
+        struct DummySigner;
+        impl Signer for DummySigner {
+            fn sign_bytes(&self, _: &[u8], _: &Address) -> Result<Signature, anyhow::Error> {
+                Ok(Signature::new_secp256k1(DUMMY_SIG.to_vec()))
```

Comment:
it's used in one place only, I don't think there's a need to make it a const. Would `new_secp256k1(vec![0u8])` work? Not a big deal though.